window/src/os/windows/window.rs

```diff
@@ -1012,12 +1014,29 @@ unsafe fn wm_nccalcsize(hwnd: HWND, _msg: UINT, wparam: WPARAM, lparam: LPARAM)
     Some(0)
 }
 
+unsafe fn maximize_button_coords(
+    hwnd: HWND,
+) -> euclid::Point2D<isize, wezterm_input_types::PixelUnit> {
+    let mut client_rect = RECT::default();
+    GetClientRect(hwnd, &mut client_rect);
+
+    let scale = GetDpiForWindow(hwnd) as f64 / 96.0;
+
+    let mut coords = mouse_coords(0);
+
+    coords.x = client_rect.right as isize - (60.0 * scale) as isize;
+    coords.y = 5;
```

Comment:
Can these hardcoded numbers be extracted from window/theme metrics? Take a look at the `get_os_parameters` function impl in this file:

https://github.com/wez/wezterm/blob/main/window/src/os/windows/window.rs#L826

Reply:
That's just center of the maximize button to hover/click on it, it will be better if I understood how to transform screen relative coordinates to client area relative coordinates. I tried to substract client_rect's values but it looks that it has 5px shift for window resizing area.

Also I use hardcoded coorinated for minimize button, but if I correctly can handle cursor position I will be able to set static variable to inform wm_nchittest that cursor pointing onto maximize button

Reply:
Also I need ability to correctly handle gnome styled circle buttons, because currently they behave like squares